templates/header.res.erb

```diff
@@ -18,6 +18,14 @@ resource <%= @name %> {
 <% end -%>
   }
 
+  startup {
+<% if @startup_parameters -%>
```

Comment:
Like `disk_parameters, I think this whole block should be hidden if no parameters are present.